node_modules/oae-principals/lib/internal/dao.js

```diff
@@ -32,6 +32,50 @@ var User = require('oae-principals/lib/model').User;
 
 var RESTRICTED_FIELDS = ['acceptedTC', 'admin:tenant', 'admin:global'];
 
+// Users have 24 hours to verify an email address
```

Comment:
Do we have a way to indicate the token was valid but expired so we can offer the ability to resend the verification email?